src/DebugCmdInfoConstants.cc

```diff
@@ -5,282 +5,221 @@
 //
```

Comment:
This file is technically generated from a script. Should I just revert the whole thing?

Reply:
yeah, excluding all generated code makes sense